main.go

```diff
@@ -2,11 +2,10 @@ package main
 
 import (
 	"errors"
-	"log"
-	"net"
-
 	"github.com/google/gopacket"
 	"github.com/google/gopacket/pcap"
+	"log"
+	"net"
```

Comment:
External package imports are last according to golinter , let's keep the code properly linted

Reply:
I'll add a Github action to run linter later